config/routes.rb

```diff
@@ -8,4 +8,6 @@
   get "/destinations", to: "destinations#index"
 
   get "/destinations/show", to: "destinations#show"
+
+  get '/cart', to: 'cart#show'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.